.jenkins/test.sh

```diff
@@ -0,0 +1,103 @@
+o!/bin/bash -f
```

Comment:
Can you include a short comment about what the arguments (if any) are for this script? It looks like there are none but it's hard for me to tell with my limited bash knowledge.

Also is this a typo?
```suggestion
#!/bin/bash -f
```

I'd really like to see CircleCI use this same test script. Someone could do this in a later PR.

Reply:
Nice catch!